### ISSUE TITLE
Content shows in editor but not on player/shared schedules

### ISSUE THREAD

**Issue by sheadarlison** [Rise-Vision/widget-html#46]
https://apps.risevision.com/editor/workspace/ddbc89db-e004-4bd5-8728-9a31b0b7af94?cid=7fa5ee92-7deb-450b-a8d5-e5ed648c575f

embed code: `<div class="social-media-wall"  data-cfg-wallid=75664></div><script async src="https://static.socialmediawall.io/w/init-wall.js"></script>`

**Comment by stulees** [comment]
@sheadarlison Have you tried removing `async` in `<script>` tag?

**Comment by olegrise** [comment]
@sheadarlison  I couldn't find what causes the problem on our side, so I looked at the embed code. I noticed that embedded content is loaded the same way in both Preview and Shared Schedules, but then in Shared Schedules it gets stuck on waiting for some internal message to make the content visible. I couldn't figure out what triggers this message and why it's not fired when embed code is loaded in Shared Schedules. 

To recap, if making this embed code working on our platform is important for us, then my recommendation is to get in touch with the company that developed that script and ask them why their widget is not rendering.

**Comment by olegrise** [comment]
@sheadarlison have you seen my comment above? I'm wondering if we can close this issue.

**Comment by sheadarlison** [comment]
@stulees I tried removing async but that didn't seem to solve the issue. Any other ideas?

@olegrise I've sent them an email.

**Comment by stulees** [comment]
@sheadarlison No other ideas and @olegrise investigation also has not turned up any problem on our end. At this point I think the company need to look at what could be the reason it does not render. Do you agree to close the issue until they provide some clarity and/or useful information? Thanks

**Comment by sheadarlison** [comment]
@stulees okay, we can close till they respond
